commands/tocfl/tocfl.py

```diff
@@ -56,10 +57,16 @@ async def tocfl_rand(
 def _create_word_embed(
     word: str, level: int, part_of_speech: str, pinyin: str
 ):
+    try:
+        chewing = to_chewing(pinyin)
+    except AssertionError:
+        chewing = "ERROR OCCURRED"
+
     embed = discord.Embed(
         title=word
     )  # ^ add description="desc" for translation
-    embed.add_field(name="Pronunciation", value=pinyin, inline=False)
+    embed.add_field(name="Pronunciation (Pinyin)", value=pinyin, inline=False)
+    embed.add_field(name="Pronunciation (Zhuyin)", value=chewing, inline=False)

```

Comment:
Then if `chewing` is `None` you should simply not display the zhuyin, e.g.:

`if chewing:  embed.add_field(name="Pronunciation (Zhuyin)", value=chewing, inline=False)`